ROADMAP.md

```diff
@@ -2,60 +2,47 @@
+| | Work to reduce time to cluster availability on vSphere and public clouds (Cluster API v1.1 Proposal) | Awaiting Proposal | TBD |
+| | Make management clusters registerable in Tanzu Mission Control | Implementation in progress | v0.11 |
+| | Introduce a UI that takes a user from installation through creating clusters to installing packages on clusters. Embed documentation and guidance throughout so that users can be successful exclusively through this tooling. | Awaiting Proposal | TBD |
+| | Expose Cluster API (capi)-provider bootstrapping details, client-side, to users when creating management and workload clusters. Provides users an understanding of where bootstrapping failures occurred. | Awaiting Proposal [(issue)](https://github.com/vmware-tanzu/community-edition/issues/2730 ) | TBD |
+|Robust Kubernetes platform | Limited-internet (including airgapped) support | Awaiting Proposal | TBD |
+| | Validate and improve end-to-end tests across all supported infrastructure providers. | Awaiting proposal | TBD |
+| Rich package library | Work with community to solicit and prioritize desired packages| n/a |ongoing |
+| | Pursue community and ISV package lifecycle and configuration integrations via Carvel and Tanzu Framework | n/a | ongoing |
+| Positioning Community Edition as leading-edge upstream for Tanzu products | Community Edition produces its own Bill of Materials (BOM), Tanzu Kubernetes Release (TKr), OCI container images, host images, and packages for CLI plugins and in-cluster software | Awaiting Proposals | TBD |
+| Application Platform | Installable package is made available containing pre-configured, minimally-viable, platform services that enable developers to run their tools and apps (contour, kpack, cert-manager, knative serving, Cartographer, etc.) | Implementation in Progress (TODO: tracker proposal for all components) | Beginning in v0.10; please see issue (in dev) |
+| | Tools and processes for package contribution, maintenance, and installation; Package maintainer documentation and guidance is live such that everyone can understand the responsibilities of being a package maintainer | Awaiting Proposal | TBD |
+| |Provide guidance on bringing new provider packages to TCE; enabling the inclusion of new infrastructure providers | Awaiting Proposal (dependent on Tanzu Kubernetes Provider implementation) | TBD |
+| |Introduce kubeapps to TCE for discovering and installation packages in Bitnami library [(issue)](https://github.com/vmware-tanzu/community-edition/issues/2418) | Awaiting Proposal | TBD |
+| Better defined LCM Cluster API extensibility points |Build the Runtime Extension SDK in Cluster API for ClusterClass patching, pre/post control plane upgrade hooks | Awaiting proposal | TBD |
+| Accelerate time to bootstrap Cluster API management clusters | Consider more efficient and resource-saving methods to bootstrap | Awaiting proposal | TBD |
```

Comment:
I don't understand this one.

Are we saying that the TCE team should execute against the `Cluster-API` library (aka, make upstream contributions) in order to make the bootstrapping of a Cluster API management cluster faster?

Or is this work that `tanzu-framework` needs to execute against in order to speed up the process of bootstrapping a management cluster? (this makes the most sense to me since they are pulling in that library and harnessing the bootstraping capabilities of Cluster API)

Or is this saying that we _completely reconsider_ using Cluster API in order to find "more efficient and resource-saving methods to bootstrap"?